libs/feather/targetoverrides.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace pins {
+    //% fixedInstance shim=pxt::getPinCfg(CFG_PIN_A8)
+    const A8: AnalogInPin;
+    //% fixedInstance shim=pxt::getPinCfg(CFG_PIN_A9)
+    const A9: AnalogInPin;
+}
```

Comment:
where are these? I can see D2 and D3 on the bottom https://learn.adafruit.com/adafruit-pybadge/pinouts

Reply:
there's also no a8/a9 in the bootloader config it seems

Reply:
D2,D3 don’t expose analogRead()

Reply:
I need to use the d2 d3 bootloader values then.

Reply:
do `D2: AnalogInPin`; these are labelled D2/D3 on the board